backend-py/src/api/endpoints/base.py

```diff
@@ -16,15 +16,17 @@ def register_api(
         url,
         defaults={pk: None},
         view_func=view_func,
-        methods=["GET", ]
+        methods=[
+            "GET",
+        ],
```

Comment:
I know black is supposed to be "don't talk about formatting" -- but if you remove the comma here black will produce better formatting (same for the one below)